projects/extension/src/background/index.ts

```diff
@@ -1,15 +1,36 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
You can avoid this by adding a `.catch((err) => console.log(err))` at the promise

Reply:
why? the promise is already handling the errors, the callback function of that catch will never be called.